tests/test_rendering.py

```diff
@@ -20,16 +20,27 @@
                                'rv:31.0) Gecko/20100101 Firefox/31.0')}
 
 
-class TestURLs(unittest.TestCase):
+class TestURIContent(unittest.TestCase):
+    """Tests for the content once HTML remplates are rendered."""
```

Comment:
nit: templates

Reply:
ahah cute. Must have been rambling about the templates.
Thanks.